marked-element.html

```diff
@@ -309,6 +309,12 @@
       this.fire('marked-render-complete', {}, {composed: true});
     },
 
+    /**
+     * Fired when the content is being processed and before it is rendered.
+     * Provides an opportunity to implement syntax highlighting, e.g with [highlightjs](https://highlightjs.org/).
```

Comment:
what does "implement syntax highlighting mean"? Unclear from just reading, add some more detail or an example

Reply:
@FredKSchott Adding an example and demo is kinda where the other PR stalled because it would require creating a custom syntax theme for this.

Reply:
Just explaining what that means in more detail is enough for me.

Reply:
well, hopefully that is better :grimacing: , I hate writing and explaining things :sweat_smile: